### PR TITLE
Manage downloads of policies.json in python

### PR DESCRIPTION
**Pull request by cdrappi**
This commit allows the user to download or update their policies.json file inside the `send-tweets.py` file. It moves the `policies.json` file to where `npm` would install it by default, and then adds all the npm crap to `.gitignore`, including that big file.

I kind of like the idea of adding the policies file to source, but it's a little weird to make a PR with 2300 lines. Commenting out line 15 would add that file to git. I just learned about this `!` business in gitignores today, and also the difference between ending in "/*" vs just "/"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "repository": "https://www.github.com/kairstenfay/yang-bot",
+  "license": "MIT",
```
Comment (cdrappi) [inline package.json:3]: also took a stab at this for you but LMK if you prefer a different one!

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Make sure you've set these for the Twitter API:
 ## Download the data
 If you'd like to update Yang policy data, make sure you have [`npm`](https://www.npmjs.com/get-npm) installed and then run:
 ```sh
-python post-tweets.py --update --no-tweet
+python post-tweets.py --update --no_tweet
```
Comment (kairstenfay) [inline README.md]: I actually prefer the hyphenated flag versus the snake_case flag here. 

Naming the flag `no-tweet` in the parser works fine for me!

**Review by kairstenfay — CHANGES_REQUESTED** [review]
Wow this is fantastic. Thank you so much for adding documentation and enhancing the developer experience here! 

I am nearly ready to merge, but I would prefer to rename the `no_tweet` flag to `no-tweet` to be more consistent with CLI tool option naming.

File: package.json (hunk 1 above)
Comment (kairstenfay) [reply package.json:3]: This is wonderful, thank you.

**Comment by kairstenfay** [comment]
> Commenting out line 15 would add that file to git. I just learned about this `!` business in gitignores today, and also the difference between ending in "/*" vs just "/"

Neat! I don't feel particularly strong about this either way, so I'm fine with leaving it as an open TODO.

**Comment by cdrappi** [comment]
@kairstenfay thanks for your review! I changed it to `--no-tweet`

**Review by kairstenfay — APPROVED** [review]
Superb! Merging now.

**Pull request merged**